makerdao/governance-portal-v2
# Use @ensdomains/ens-avatar for resolving avatars, remove image fetchi…

### What does this PR do?

Use @ensdomains/ens-avatar for resolving avatars

Remove image fetching endpoint

### Steps for testing:

Go to poll at `/polling/QmY85FT6#vote-breakdown`

See that some ENS avatars are resolving

### Screenshots (if relevant):
<img width="575" alt="Screen Shot 2022-08-23 at 12 34 13 PM" src="https://user-images.githubusercontent.com/5225766/186137123-0d87360b-f8a3-4b45-9bdb-7cf92d342a57.png">
